pkg/distro/rhel/rhel10/options.go

```diff
@@ -1,108 +0,0 @@
-package rhel10
-
-import (
-	"fmt"
-
-	"slices"
-
-	"github.com/osbuild/images/internal/common"
-	"github.com/osbuild/images/pkg/blueprint"
-	"github.com/osbuild/images/pkg/customizations/oscap"
-	"github.com/osbuild/images/pkg/distro"
-	"github.com/osbuild/images/pkg/distro/rhel"
-	"github.com/osbuild/images/pkg/policies"
-)
-
-// checkOptions checks the validity and compatibility of options and customizations for the image type.
-// Returns ([]string, error) where []string, if non-nil, will hold any generated warnings (e.g. deprecation notices).
-func checkOptions(t *rhel.ImageType, bp *blueprint.Blueprint, options distro.ImageOptions) ([]string, error) {
```

Comment:
This needs double checking that really all options are ported over to "generic"

Reply:
We currently do not port the checks `UnsupportedPartitioningModes` - this is only needed in rhel-9,8, when we port the images to the generic distro this will happen there.